mlflow/projects/databricks.py

```diff
@@ -58,6 +58,11 @@ def _get_databricks_run_cmd(dbfs_fuse_tar_uri, entry_point, parameters):
     project_dir = os.path.join(DB_PROJECTS_BASE, tar_hash)
     mlflow_run_arr = list(map(shlex_quote, ["mlflow", "run", project_dir, "--new-dir",
                                             "--entry-point", entry_point]))
+    # TODO: we need this check since run ID may be undefined e.g. if we launch a Databricks run
```

Comment:
It seems safe to always condition extension on existence, I don't know if we need all this TODO. We can just keep the if statement.